static/src/javascripts/bootstraps/enhanced/gallery.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+import qwery from 'qwery';
+import config from 'lib/config';
+import mediator from 'lib/mediator';
+import Component from 'common/modules/component';
+import { initTrails } from 'bootstraps/enhanced/trail';
+
+const transcludeMostPopular = (): void => {
+    const mostViewed = new Component();
+    const container = qwery('.js-gallery-most-popular')[0];
+
+    mostViewed.manipulationType = 'html';
+    mostViewed.endpoint = '/gallery/most-viewed.json';
+    mostViewed.ready = (): void => {
+        mediator.emit('page:new-content', container);
+    };
```

Comment:
Neither of the `mediator` events are listened for anywhere. Maybe we can delete them and drop the dependency? Or do you reckon consistency with (certain) other bootstraps is more important than dropping dead code?

Reply:
oh no, good catch, thanks!

Reply:
in fact nothing seems to listen for `page:new-content`?

Reply:
That's true. And it's pointlessly emitted in 7 places across the application

Reply:
will remove them in #17940 first. prob worth an event audit I reckon!